capstone/src/components/App/App.jsx

```diff
@@ -1,28 +1,56 @@
 import './App.css';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import axios from 'axios';
 import Visitor from '../Visitor/Visitor';
-
+import User from '../User/User';
 // CONTEXTS
-// import LoginContext from '../Contexts/LoginContext';
+import UserContext from '../../Contexts/UserContext';
 // import SignUpContext from '../Contexts/SignUpContext';
 
+export async function getUserInfo(userId) {
+  const values = { objectId: userId };
+  const baseURL = 'http://localhost:3001';
+  return axios.post(`${baseURL}/visitor/user`, values);
+}
+
 export default function App() {
   // TODO: Handle state with useContext and useMemo
-  const [isLoggedIn, setIsLoggedIn] = useState(localStorage.getItem('current_user_id') !== null);
-
+  const [isLoggedIn, setIsLoggedIn] = useState(localStorage.getItem('current_user_id') != null);
+  const [userData, setUserData] = useState({});
+  async function retrieveUserData() {
+    try {
+      const data = await getUserInfo(localStorage.getItem('current_user_id'));
+      setUserData(data);
+    } catch (err) {
+      console.log(err);
+    }
+  }
```

Comment:
It might be a good time to think about **readability** and **maintainability**.

* Imagine you're brand new to this codebase - is the code easy to understand, and is everything where you would expect it to be?
* Imagine that you're going to commit a dozen more features and write 5000 more lines of code. Is it clear where that code will go, and will it still be easy to understand afterward?